src/DevTo.Api/UsersService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Forem.Api.Models;
+
+namespace Forem.Api
+{
+	public class UsersService : ApiService, IUsersApi
+	{
+		public UsersService(Uri baseUri, HttpClient httpClient) : base(baseUri, httpClient) { }
+
+		public Task<User> GetUserAsync(int id) => GetAsync<User>($"/api/users/{id}");
+
+		public Task<User> GetUserAsync(string username)
+			=> GetAsync<User>("/api/users/by_username", new Dictionary<string, object>
+				{
+				  { "url", username },
```

Comment:
I'm not sure if the parameter is actually "url" though I also can't double check at the moment as the DEV API docs seem to be down.